src/components/desktop/Header.js

```diff
@@ -60,21 +61,24 @@ const StyledHeader = styled.header`
 `
 
 // TODO: Clean up file structure (i.e. logo_white.svg) from ./public
-const Header = () => (
-  <StyledHeader>
-    <img src="/logo_white.svg" alt="Falling Fruit Logo" />
-    <nav>
-      <ul>
-        <li className="active">Map</li>
-        <li>About</li>
-        <li>
-          <button>
-            <User /> Login
-          </button>
-        </li>
-      </ul>
-    </nav>
-  </StyledHeader>
-)
+const Header = () => {
+  const { t } = useTranslation()
+  return (
+    <StyledHeader>
+      <img src="/logo_white.svg" alt="Falling Fruit Logo" />
+      <nav>
+        <ul>
+          <li className="active"> {t('Map')}</li>
+          <li> {t('About')}</li>
```

Comment:
Remove extra space before these two (not the button):

`<li>{t('Map')}</li>`

`<li>{t('About')}</li>`